robo_soccer_platform/lib/robo_soccer_platform_web/live/game_dashboard/components.ex

```diff
@@ -72,13 +72,24 @@ defmodule RoboSoccerPlatformWeb.GameDashboard.Components do
 
     ~H"""
     <div class="flex flex-col flex-1 gap-2">
-      <div class="ml-auto">
-        <.time_left seconds={@seconds_left} />
-      </div>
       <div class="grid grid-flow-col auto-cols-fr w-full gap-2">
         <div class="flex flex-col flex-1 gap-4">
           <.cooperation_metrics red_cooperation_metric={@red_cooperation_metric} green_cooperation_metric={@green_cooperation_metric} />
-          <.directions_and_score red_direction={@red_direction} green_direction={@green_direction} red_goals={@teams["red"].goals} green_goals={@teams["green"].goals} />
+
+          <div class="flex gap-8">
+            <.time_left
```

Comment:
this component changes it's width according to the length of the text in it, shouldn't be like that